src/platform/react-hooks/src/hooks/usePresence.ts

```diff
@@ -1,4 +1,4 @@
-import * as Ably from '../../../../../ably.js';
+import type * as Ably from 'ably';
```

Comment:
Not related to this PR, but I've noticed that v2 branch is importing 'ably' incorrectly in react hooks.
See opened issue for this: https://github.com/ably/ably-js/issues/1619.

This PR correctly resolved merge conflicts for 'usePresence.ts' and `usePresence.test.tsx`, so it has `import type * as Ably from 'ably';`.
I believe all other react hook files also should import in the same way